pages/services/tree-trimming.tsx

```diff
@@ -13,22 +13,21 @@ export default function TreeTrimming() {
   return (
     <>
       <NextSeo
-        title='Residential & Commercial Tree Trimming Service'
+        title='Residential & Commercial Tree Trimming Service in Silver Spring, MD'
         description="We provide residential and commercial tree trimming services in Silver Spring, Maryland. Keep your trees looking great and healthy with Kuhn's Tree Service. Call us today!"
       />
 
       <Hero
         backgroundImage={heroBg}
-        text='Tree Trimming in Silver Spring, Maryland'
+        text='Residential & Commercial Tree Trimming Services'
+        alt='Person cutting a tree branch with a chainsaw.'
       />
 
       <SectionWrapper className='lg:max-w-3xl'>
         <div className='flex flex-col'>
           <h2 className='pt-5 pb-10'>Tree Trimming from Kuhn's Tree Service</h2>
           <p className='pb-5'>
-            When you work with Kuhn's Tree Service, you can expect high-quality
-            tree trimming services and professionalism unparalleled in the
-            industry. For more than 25 years, we've been helping residents and
+            When you work with Kuhn's Tree Service, you can expect high-quality residential and commercial tree trimming services and professionalism unparalleled in the industry.For more than 25 years, we've been helping residents and
```

Comment:
There's some grammar mistakes here. Looks like a run-on sentence and missing a space after the period.

Reply:
Space added after the period.